lib/talis-chart/calculate-resource-quota.ts

```diff
@@ -204,6 +208,10 @@ export function calculateResourceQuota(
       const memoryRequest = getResourceRequest(workload, container, "memory");
       workloadCpu += cpuToMillicores(cpuRequest);
       workloadMemory += memoryToBytes(memoryRequest);
+      if (appMeshEnabled) {
+        workloadCpu += appMeshCpu;
+        workloadMemory += appMeshMemory;
+      }
```

Comment:
This would add app mesh sidecar per container in the pod, while Envoy and X-ray are just additional containers in the Pods. So, this `if` should be outside of the `for` loop, i.e. count the once per workload.

Reply:
Ahh wasn't sure exactly where it needed to be in the loop so made a best guess